storage_backend/models/storage_backend.py

```diff
@@ -5,13 +5,54 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 import base64
+import functools
+import inspect
 import logging
+import warnings
 
 from odoo import _, fields, models
 
 _logger = logging.getLogger(__name__)
 
 
+# TODO: useful for the whole OCA?
```

Comment:
yes this should be move in a generic tool !

Reply:
suggestions?
Shall we embrace the proposal by Pedro to have an all in one `odoo-oca-lib`?

Reply:
@simahawk Personally I'm against a all in one lib. I prefer a small (even very small) lib with a dedicated scope than one with tons of unrelated stuffs.